silence/silence.go

```diff
@@ -853,13 +890,19 @@ func (s *Silences) query(q *query, now time.Time) ([]*pb.Silence, int, error) {
 			}
 		}
 		if !remove {
-			resf = append(resf, cloneSilence(sil))
+			resf = append(resf, shallowClone(sil))
 		}
 	}
 
 	return resf, s.version, nil
 }
 
+// shallowClone returns a shallow copy of a silence.
+func shallowClone(sil *pb.Silence) *pb.Silence {
```

Comment:
This is the old `cloneSilence` function. Renamed `cloneSilence` to `shallowClone` since we now have a deep copy function called `Clone`.